process/graduation_criteria.md

```diff
@@ -0,0 +1,118 @@
+# Azure Container Upstream Graduation Criteria v0.1
+The Azure Container Upstream Graduation Criteria owes much of its basis on two existing documents,
+[CNCF Graduation Criteria](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/graduation_criteria.adoc) 
+and [.NET Foundation Project Maturity Profile](https://github.com/dotnet-foundation/project-maturity-model/blob/65fdfa43d7e68845bd65638c378cf834d7158b90/maturity-profiles.md). The goal of this document is to describe the process and 
+minimum bar for each increasing level of project maturity to both provide a
+goal for the project maintainers, but also to set expectations for consumers of these
+projects.
+
+## Process
+
+Each project in Container Upstream (AzCU) has an associated maturity level. A proposed AzCU project 
```

Comment:
upstream without containers. Some of the work we do is beyond "Containers"

Reply:
Agreed that what we do is beyond "containers" but "container" is part of our team name :)

Reply:
We could change this to `Cloud Native Upstream`.